docs/DeveloperGuide.md

```diff
@@ -275,31 +275,35 @@ The target users...
+| `*`     | user                     | group delete patients by category or tag                               | save time when removing large amount of patient records               |
+| `* *`   | user                     | search for specialists by their speciality                             | refer patients to the right experts                                   |
+| `*`     | user with poor eyesight  | customise font size and style of the App                               | easily read the information on the screen                             |
+| `*`     | user who is colour blind | customise colour schemes of the App                                    | meet my accessibility needs                                           |
+| `*`     | user who is a fast typer | perform all tasks will the CLI rather than the GUI                     | be more efficient in managing records                                 |
+| `* *`   | user who is impatient    | add new keyboard shortcuts to the App                                  | save time when performing frequently repeated tasks                   |
+| `*`     | lazy user                | import App data previously stored as a .CSV file directly into the App | save time on setting up the application                               |
+| `*`     | lazy user                | export App data                                                        | share the data easily when required                                   |
+| `* * *` | user                     | save the App data                                                      | retain data for future reference                                      |
+| `* * *` | user                     | load previously saved App data                                         | examine past records/information                                      |
+| `*`     | experienced user         | manipulate the save file directly                                      | make quick changes to data without having to open the application     |
+| `* * *` | user                     | add a new specialist to the records                                    | keep track of details of new specialists patients can be referred  to |
+| `* *`   | user                     | edit details of existing specialists                                   | make sure the specialist details are up to date                       |
+| `* * *` | user                     | delete specialist details                                              | delete records of specialists I no longer work with                   |
+| `* *`   | user                     | access specialist details                                              | provide relevant details to the patient I am reffering                |
+| `* *`   | user                     | search for specialists by location                                     | refer patients to convenient specialists to minimise time             | 
```

Comment:
Perhaps it would be better to insert your additions according to how they are ordered in our existing Excel sheet, so that our use cases are grouped by category. (The use cases you have added might be more appropriately placed after `group delete patients by category of tag` and before `search for specialists by their speciality`,)